charts/warpstream-agent/templates/deployment.yaml

```diff
@@ -63,6 +63,12 @@ spec:
               port: http
             initialDelaySeconds: 10
             periodSeconds: 5
+          livenessProbe:
+            httpGet:
+              path: /v1/status
+              port: http
+            initialDelaySeconds: 10
+            periodSeconds: 5
```

Comment:
Whats the default number of allowed failures here?

Reply:
3. You think it's too aggressive? I tried deploying an agent on minikube and it worked fine but it was not talking to a blob storage.

Reply:
5s timeout? Thats probably fine tbh. Could do 5 if we're really paranoid with 5s timeout

Reply:
putting 5, it's cheap